### ISSUE TITLE
Indentation of closure inside multiline function call

### ISSUE THREAD

**Issue by maurobringolf** [WordPress-Coding-Standards/WordPress-Coding-Standards#899]
I am getting contradicting error messages around a closure used in a multiline function call:
```php
<?php
array_map( function( $item ) {
    echo 'hello';
    },
    $arr
);
```
I get this: "Line indented incorrectly; expected 0 tabs, found 1". If I do not indent the closing brace it says "Multi-line function call not indented correctly; expected 4 spaces but found 0". What would be the way to format this?

Thanks for any help

**Comment by jrfnl** [comment]
Hi Mauro, thanks for opening the separate issue. If you don't mind, I'll remove your previous post from the other issue so as not to confuse that one.

Regarding this example: you are mixing the syntax for a single-line and multi-line function call.
A single line function call would have all the parameters on the same line.
A multi line function call would have all the parameters each on their own line.
Your example has the first parameter set up as a single line, while the second is set up as if it was a multi-line call.

Mixing these styles is what causes the sniff to throw the error. Using either one of the styles consistently, gets rid of the error.

```php
// Multi-line
array_map(
	function( $item ) {
		echo 'hello';
	},
	$arr
);

// Single line
array_map( function( $item ) {
	echo 'hello';
}, $arr );

```

**Comment by maurobringolf** [comment]
Oh wow, thank you for such quick feedback! Thank you for the clear explanation, my bad.

**Comment by jrfnl** [comment]
@maurobringolf No problem, glad it helped.
